imports/api/regions/regions.js

```diff
@@ -0,0 +1,14 @@
+import { Meteor } from 'meteor/meteor';
+import { Mongo } from 'meteor/mongo';
+
+// ======== DB-Model: ========
+// _id              -> ID
+// name             -> String
+// loc              -> Geodata {type:Point, coordinates: [long, lat]}  (not lat-long !)
+// timeZone         -> String,  ex: "UTC+01:00"
+// courseCount      -> Number of courses in that region, calculated field
+// futureEventCount -> Number of future events in that region, calculated field
+// ===========================
```

Comment:
Document the `featuredGroup` field here.

Taking the wider perspective, you could consider keeping a spearate `FeaturedGroups` collection instead of attaching the info to the regions. That would be more flexible. For a the current scope it seems like overkill though.